webrender/src/device.rs

```diff
@@ -639,6 +639,36 @@ impl<T> GpuProfiler<T> {
     }
 }
 
+pub struct GpuMarker(());
```

Comment:
No, since it's unsafe to create an instance of it without calling `GpuMarker::new()`